app/recommender_engine_stub.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
```

Comment:
This is just a side-note - I'm not the biggest fan of importing some package under another name if it isn't in order to avoid name clashing.

Reducing number of characters from 5 to 2 is not the biggest win when contrasted with readability.
(Though, I understand that a lot of  people imports `numpy` as `np`, so I won't fight too here)

Reply:
Yes, it can look like an unnecessary characters reduction but this convention has a point: during constructing complicated vectorized calculations (that corresponds to e.g. some math equations) there occur nesting of numpy functions.
And yes, this is a way how numpy is imported absolutely everywhere.